cou/utils/openstack.py

```diff
@@ -47,7 +47,7 @@
 DATA_PLANE_CHARMS = ["nova-compute", "ceph-osd", "swift-proxy", "swift-storage"]
 
 # https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#list-the-upgrade-order
-UPGRADE_ORDER = [
+OFFICIAL_UPGRADE_ORDER = [
```

Comment:
Nit: consider changing it to `UCA_UPGRADE_ORDER` or `OPENSTACK_SERVICES_UPGRADE_ORDER` (see https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#upgradable-services)

Reply:
on a second note, I found that the `UPGRADE_ORDER` does not follow what's in the documentation: (e.g. `rabbitmq-server` is not an openstack services) and should not be in the `UPGRADE_ORDER`